keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -228,12 +241,20 @@ the uses and implications of any populators they chose to install.
 
 ### Test Plan
 
-The minimal test for this feature is to create a PVC with a data source
+The test for this feature gate is to create a PVC with a data source
```

Comment:
snapshot -> VolumeSnapshot to be precise

Reply:
Will fix